src/parts/Sidebar/SidebarContent/Navigation/index.tsx

```diff
@@ -149,24 +188,18 @@ const Navigation = ({
                   { 'dark:hover:bg-white': process.env.REACT_APP_RELAY_CHAIN_NAME === KUSAMA },
                   { 'dark:hover:bg-opacity-10': process.env.REACT_APP_RELAY_CHAIN_NAME === KUSAMA }
                 ),
-              'group',
-              'flex',
-              'items-center',
-              'px-2',
-              'py-2',
               onSmallScreen ? 'text-base' : 'text-sm',
```

Comment:
@tomjeatt 
This could be included in `textClasses`.

Reply:
@crypto-engineer as above